docs/register_client.md

```diff
@@ -0,0 +1,64 @@
+## Prep your environment
+1.  ensure rhel9 repos are enabled and snapshotted
+```
+make repos-import-rhel9
+```
+2. 
+```
+go run cmd/external-repos/main.go nightly-jobs
+```
+3.  Let those repositories snapshot
+
+4.  Note that the current instructions do not work with content_guards enabled.  If `custom_repo_content_guards` is set to true in config.yaml, content access will be denied for custom repositories.
+
+## Configure firewall
+ You might need to adjust your firewall to allow access.  If you run into connection issues, you can set your default firewall zone to "trusted":
+```bash
+sudo firewall-cmd --set-default-zone=trusted
+```
```

Comment:
can you also mention this command is for linux? and is there a more specific explanation about what to allow access to? mainly for the non-linux people who would have to do this differently.